app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, optional: true
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you use` after_save :update_likes_counter` 👍